docs/5.x/changelog.md

```diff
@@ -23,6 +23,17 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 5.x Releases
 
+### 5.3.2
+
+#### Bugfixes
+
+* Fix an issue with cluster expansion when applying taints via app.yaml.
+* Fix an issue with labelling of packages, which could prevent upgrades from completing.
```

Comment:
Isn't it "labeling"? (single "l" in the middle)

Reply:
Oops... Canadian / British spelling :)